### PR TITLE
Chore/ts strict backend

### PR DESCRIPTION
**Pull request by EmiM**

### CODE REVIEW

File: packages/backend/src/registration/functions.ts [hunk 1]
```diff
@@ -194,6 +200,14 @@ export const sendCertificateRegistrationRequest = async (
 
     const parsedCsr = await loadCSR(userData.csr)
     const username = getReqFieldValue(parsedCsr, CertFieldsTypes.nickName)
+    if (!username) {
+      log.error(`Could not parse certificate for field type ${CertFieldsTypes.nickName}`)
+      return {
+        // Should be internal server error code 500
```
Comment (Kacper-RF) [inline packages/backend/src/registration/functions.ts:206]: Maybe create a task for it, to not forget about it

File: packages/backend/src/storage/ChannelsAccessController.ts [hunk 2]
```diff
@@ -53,7 +56,7 @@ export const createChannelAccessController = (peerId) => {
       return ''
     }
 
-    static create(_orbitdb, _options) {
+    static create(_orbitdb: OrbitDB, _type = type, _options: any) {
```
Comment (Kacper-RF) [inline packages/backend/src/storage/ChannelsAccessController.ts:59]: Any possibility to replace `any` another more specyfic type in this case ?

File: packages/backend/src/storage/storage.test.ts [hunk 3]
```diff
@@ -53,48 +42,48 @@ let tmpAppDataPath: string
...
-let channel: PublicChannelStorage
+let channel: PublicChannel
 let alice: Identity
 let john: Identity
 let message: ChannelMessage
-let channelio: PublicChannelStorage
+let channelio: PublicChannel
 let filePath: string
-let utils
+let utils: any
```
Comment (Kacper-RF) [inline packages/backend/src/storage/storage.test.ts:55]: Also here

File: packages/backend/src/storage/storage.test.ts [hunk 4]
```diff
@@ -308,8 +307,8 @@ describe('Certificate', () => {
     expect(spyOnUpdatePeersList).toBeCalled()
   })
 
-  it.each(['write, replicate.progress'])('The message is verified valid on "%s" db event', async () => {
-    const eventName = 'write'
+  it.each(['write', 'replicate.progress'])('The message is verified valid on "%s" db event', async (eventName: string) => { // 'replicate.progress'
+    // const eventName = 'write'
```
Comment (Kacper-RF) [inline packages/backend/src/storage/storage.test.ts:311]: Leftover?

**Review by Kacper-RF — COMMENTED** [review]
Minor comments, but also i think it will be great that someone else look at this code, because is huge PR

File: packages/backend/src/storage/ChannelsAccessController.ts (hunk 2 above)
Comment (EmiM) [reply packages/backend/src/storage/ChannelsAccessController.ts:59]: AccessControllers.create also has 'any' as options type so I don't think so.

File: packages/backend/src/registration/functions.ts (hunk 1 above)
Comment (EmiM) [reply packages/backend/src/registration/functions.ts:206]: https://github.com/TryQuiet/quiet/issues/1514

**Review by Kacper-RF — APPROVED** [review]
LGTM

**Pull request merged**